jjb/dynamic/SVT_Conformance.yml

```diff
@@ -0,0 +1,169 @@
+          command and copy the results.
+        name: ENABLE_PBENCH
+    - bool:
+        default: false
+        description: Use the ProxyCommand to access the OpenShift masters on private
+          networks in the lab.
+        name: USE_PROXY
+    - string:
+        default: ''
+        description: The name to use in the ssh ProxyCommand.
+        name: PROXY_USER
+    - string:
+        default: ''
+        description: The server to use in the ssh ProxyCommand.
+        name: PROXY_HOST
+    - string:
+        default: ''
+        description: ''
+        name: GITHUB_TOKEN
+    - bool:
+        default: false
+        description: ''
+        name: OCP_4
+    - bool:
+        default: false
+        description: ''
+        name: CONTAINERIZED
+    project-type: freestyle
+    properties:
+    - build-discarder:
+        artifact-days-to-keep: -1
+        artifact-num-to-keep: -1
+        days-to-keep: -1
+        num-to-keep: 7
+    - raw:
+        xml: |
+          <hudson.plugins.disk__usage.DiskUsageProperty plugin="disk-usage@0.28" />
+    - raw:
+        xml: |
+          <com.dabsquared.gitlabjenkins.connection.GitLabConnectionProperty plugin="gitlab-plugin@1.5.3">
+          <gitLabConnection />
+          </com.dabsquared.gitlabjenkins.connection.GitLabConnectionProperty>
+    - raw:
+        xml: |
+          <org.jenkinsci.plugins.ZMQEventPublisher.HudsonNotificationProperty plugin="zmq-event-publisher@0.0.5">
+          <enabled>false</enabled>
+          </org.jenkinsci.plugins.ZMQEventPublisher.HudsonNotificationProperty>
+    - raw:
+        xml: |
+          <com.synopsys.arc.jenkins.plugins.ownership.jobs.JobOwnerJobProperty plugin="ownership@0.11.0">
+          <ownership>
+          <ownershipEnabled>true</ownershipEnabled>
+          <primaryOwnerId>mifiedle</primaryOwnerId>
+          <coownersIds class="sorted-set" />
+          </ownership>
+          </com.synopsys.arc.jenkins.plugins.ownership.jobs.JobOwnerJobProperty>
+    - raw:
+        xml: |
+          <com.sonyericsson.rebuild.RebuildSettings plugin="rebuild@1.27">
+          <autoRebuild>false</autoRebuild>
+          <rebuildDisabled>false</rebuildDisabled>
+          </com.sonyericsson.rebuild.RebuildSettings>
+    - raw:
+        xml: |
+          <hudson.plugins.throttleconcurrents.ThrottleJobProperty plugin="throttle-concurrents@2.0.1">
+          <maxConcurrentPerNode>0</maxConcurrentPerNode>
+          <maxConcurrentTotal>0</maxConcurrentTotal>
+          <categories class="java.util.concurrent.CopyOnWriteArrayList" />
+          <throttleEnabled>false</throttleEnabled>
+          <throttleOption>project</throttleOption>
+          <limitOneJobWithMatchingParams>false</limitOneJobWithMatchingParams>
+          <paramsToUseForLimit />
+          </hudson.plugins.throttleconcurrents.ThrottleJobProperty>
+    publishers: []
+    triggers: []
+    wrappers:
+    - ansicolor:
+        colormap: xterm
```

Comment:
For conformance in 4.1 we were running openshift-tests run openshift/conformance directly.   I'll put up a 
PR to switch the wrapper script to the new test binary.

Reply:
Sounds good, thanks @mffiedler.